assets/react/controllers/Incident/TaskContext.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState } from "react";
+import { Spin } from "antd";
+import TaskStatusBadge from "../../../../js/app/dashboard/components/TaskStatusBadge";
+import { money } from "./utils";
+
+async function _fetchTaskContect(id) {
+  const httpClient = new window._auth.httpClient();
+  return await httpClient.get(
+    window.Routing.generate("api_tasks_get_task_context_item", { id }),
+  );
+}
+
+function formatContext({ delivery, order }) {
```

Comment:
that can be a React component, for example `Context` or `ContextDetails`

Reply:
I hesitated to extract it into a component. But I don't think we'll need it elsewhere, and I wanted to avoid cluttering the codebase with too many small JS files.

Reply:
It does not need to be in a separate file 😉 If you rename this function to start with a capital letter it will be already a component